src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -299,6 +302,60 @@ class guild_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_GUILD_INFO_EXTRA_INFO, guild->GetInfo().c_str()); // Extra Information
         return true;
     }
+
+    static bool HandleGuildListCommand(ChatHandler* handler, char const* /*args*/)
+    {
+        auto const& guildStore = sGuildMgr->GetGuildStore();   // map<uint32, unique_trackable_ptr<Guild>>
+
+        handler->SendSysMessage("|cff00ff00=== Guild list ===|r");
+        handler->SendSysMessage("|cff00ffffID | Name | GM | Created | Members | Lv | Bank(g)|r");
+
+        constexpr uint64 COPPER_PER_GOLD = 10000ULL;           //  1 gold = 10 000 copper
+
+        for (auto const& pair : guildStore)
```

Comment:
Use C++17 structured bindings here
```suggestion
        for (auto const& [id, g] : guildStore)
```